etna/packages/etna-js/package.json

```diff
@@ -39,6 +39,7 @@
     "redux-mock-store": "^1.5.4",
     "redux-thunk": "^2.3.0",
     "regenerator-runtime": "^0.13.5",
+    "reselect": "^3.0.1",
```

Comment:
In one of the weird quirks of how `etna-js` works, you'll need to add this same dependency to Metis and Timur's `package.json`. I get this error in Metis when I load the root view:

```
Uncaught Error: Cannot find module "reselect"
```

And blank page -- nothing loads.

Reply:
Here is a comment thread about the issue...we've been bitten by it before:

https://github.com/mountetna/monoetna/pull/145#discussion_r524530334